Changes

```diff
@@ -1,13 +1,24 @@
 Release history for Zonemaster component Zonemaster-LDNS
 
+
+3.0.0 2019-11-15
+
+ [API change]
+ - Support for Ed25519 (algorithm 15) added (#85, #51, #84)
```

Comment:
This should be labelled as an addition rather than a change. Or maybe I'm missing something?

Reply:
Well, an addition is a change. I changed to "Changed".

Reply:
Not in the semver sense. In this context "change" means breaking change and "addition" means it's backwards compatible. The addition of Ed25519 is backwards compatible, but the version bump of libldns is not.

Reply:
I seem to agree with @matsduf. If we go by @mattias-p logic, then all changes file should be re-written with same option?

Reply:
I think it would be good if we defined a template for the Changes files. It should not be too complex so that it is hard to use.